src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
I created a project (standalone application) that uses the SDK to create HDInsight clusters.
The Build Properties for the project was configured with "**Treat warnings as errors**" set to "**None**".
Please see corresponding image below.

![image](https://cloud.githubusercontent.com/assets/15322499/19765335/60a96a10-9c65-11e6-8bcc-677f6e4ddcda.png)


I changed the code to reference **DefaultStorageAccountKey** in the ClusterCreateParameters; and I see the error below (regardless of the setting above).
The error goes away when I change it back to use **DefaultStorageInfo** (shown commented in the image below)

![image](https://cloud.githubusercontent.com/assets/15322499/19765455/190fb49c-9c66-11e6-9b63-c49c8a465a17.png)


@markcowl , yes ... we have the corresponding powershell changes ready. I will send out  a PR for the powershell changes soon. The SDK code was compiled with 'TreatWarningsAsErrors'. From the above, it looks like that setting takes effect even in cases, where the library is referenced by other projects. The referencing project does not seem to compile when the obsoleted fields are referenced.

@matt1883 -- we dont throw an exception when deprecated properties are used -- the compile error (with the actionable error message) was meant to handle it.